src/util/measurementUncertaintyTool.js

```diff
@@ -0,0 +1,66 @@
+import Decimal from 'decimal.js';
+
+// Run node version with 14.18.3
+/**
+ * Custom measurement uncertainty rounding rules.
+ * @param  {number} inputValue            measured value from annotation
+ * @param  {number} uncertaintyValue      uncertainty from pixel diagonal
+ */
+
+/**
+ * Rounding
+ */
+
+function roundValue(inputValue, uncertaintyValue) {
+  const value = new Decimal(inputValue);
```

Comment:
Can we rename this variable so it is more self explanatory? i.e. decimalInputValue